scraper.rb

```diff
@@ -35,5 +34,5 @@
 
   # puts record.inspect
   ScraperWiki.save_sqlite(['council_reference'], record)
-  puts "Saving " + council_ref
+  puts "Saving " + council_ref.strip()
```

Comment:
If `council_ref` needs whitespace remove why don't you do that before the data is saved to the database?